apps/tutorial/public/docs/5-builtins/3-htmlSafe/prose.md

```diff
@@ -1,6 +1,6 @@
 Glimmer, by default, protects you against [Cross-Site-Scripting][xss] attacks, where an attack may try to get malicious code rendered on a page to scrape or send information from other user sessions.
 
-The `htmlSafe` utility is used when you have input that either _know_ to be safe, or you can be reasonably certain that rendering untrusted input will have no negative consequence (for example, maybe untrusted input is ran through a sanitizer (such as [PurifyDOM](https://github.com/cure53/DOMPurify)).
+The `htmlSafe` utility is used when you have input that you either _know_ to be safe or are reasonably certain that rendering any malicious input will have no negative consequence (for example, maybe the input is already safely handled by running it through a sanitizer (such as [PurifyDOM](https://github.com/cure53/DOMPurify)).
```

Comment:
```suggestion
The `htmlSafe` utility is used when you have an input that you either _know_ to be safe or are reasonably certain that rendering any malicious input will have no negative consequence (for example, maybe the untrusted input is already safely handled by running it through a sanitizer (such as [PurifyDOM](https://github.com/cure53/DOMPurify -- making the input trusted and safe)).
```

This is certainly an improvement in grammar -- what do you think about this suggestion? 

The main thing I want clear is that `htmlSafe` doesn't make string safe, it declares that a human has determined a string to be safe.

Reply:
Please look at the most recent commit and let me know if that's better.

Reply:
very nice! thank you!